website/src/data/tweets.tsx

```diff
@@ -30,7 +30,7 @@ const TWEETS: TweetItem[] = [
       </>
     ),
     showOnHomepage: true,
-    githubUsername: 'markerikson',
+    githubUsername: 'acemarke',
```

Comment:
looks to me the github username was correct: https://github.com/markerikson/

Reply:
my bad, i was confusing with twitter handles, reverted thx! 🙏